src/main/java/org/terasology/launcher/gui/javafx/SettingsController.java

```diff
@@ -147,13 +143,14 @@ protected void cancelSettingsAction(ActionEvent event) {
 
     @FXML
     protected void saveSettingsAction(ActionEvent event) {
+        // TODO: Clean up
         // save job
-        final JobItem jobItem = jobBox.getSelectionModel().getSelectedItem();
-        launcherSettings.setJob(jobItem.getJob());
+        //final JobItem jobItem = jobBox.getSelectionModel().getSelectedItem();
```

Comment:
@praj-foss that's what you talked about, right? Loading the last selected job and version from the settings on launcher startup needs to be enabled again. This could be good follow up to start with.

Reply:
Alright then, it'll be resolved in an upcoming PR for better settings-file operations :slightly_smiling_face: